.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -5,38 +5,106 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
-        Please make sure to fill out the entire form below,
-        providing as much context as you can in order to help us
-        triage and track down your bug as quickly as possible.
-
-        Before filing a bug, please be sure you have searched through
```

Comment:
I see this part was removed, but I still think is useful to have it

Reply:
Agreed :) 
Added back in https://github.com/open-telemetry/opentelemetry-js/pull/5086/commits/9dc41da9e90d234fb8ee5b5d8e5773797dbb2b7c